rules/docker-telemetry.mk

```diff
@@ -26,6 +26,7 @@ endif
 $(DOCKER_TELEMETRY)_CONTAINER_NAME = telemetry
 $(DOCKER_TELEMETRY)_RUN_OPT += --privileged -t
 $(DOCKER_TELEMETRY)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
+$(DOCKER_TELEMETRY)_RUN_OPT += -v /var/run/dbus:/var/run/dbus:rw
```

Comment:
Can you please share the reason, why telemetry would need rw?

Reply:
Both mgmt-framework and telemetry use host services (e.g. image management), and they will read read-write access to be able to communicate with the host.

Reply:
I get that mgmt-FW's requirement to do some host level services, like image-mgmt, config-reload, reboot ...

What is the telemetry's requirement  ? What would it need to run at host ?
Is it some gNMI related requirement ?

Can you please add the reason  for *telemetry* ?

Reply:
Telemetry container supports gNOI as well (Operations). A user can run image management/reboot commands using gNOI, therefore this is provided to support this use case.